connexta/admin-console-beta
# DDF-2773: Updated Frontend Error Codes

#### What does this PR do?
Removes some error codes that no longer exist and fixes some others.

#### Who is reviewing it?
@tbatie 

#### How should this be tested?
Passing build. Or cause an INVALID_URL or INVALID_URI error to appear anywhere in the frontend and ensure the generic error isn't returned.
